### ISSUE TITLE
Current hierarchy is not showing correct labels for place entities

### ISSUE THREAD

**Issue by kintopp** [culturesofknowledge/emplaces#3]
Currently shows:
```
+ {Name unknown}
  + http://emplaces.data.example.org/Opole_ADM2
    + http://emplaces.data.example.org/Opole_ADM3
     + City of Opole
```
Should instead show:
```
+ Poland
  + Opolskie
    + Opole 
      + Opole
```
Need to clarify if this is an issue with the RDF data or an issue with the mapping.

**Comment by kintopp** [comment]
Almost correct, now shows:

```
+ Poland
  + Opole Voivodeship (ADM1)
    + Opole (ADM2)
      + Opole (ADM3)
        + City of Opele
```

But the ADM1 etc. is in the RDF data. e.g. [Line 1162](https://github.com/culturesofknowledge/emplaces/blob/19fe2545328480fbb2d32791547caa23f3a27dc9/models/20180410-opole-example-data.ttl#L1162).

**Comment by gklyne** [comment]
Yes indeed.  I thought it would be useful at this stage to have separate labels for the administrative regions.   Note that we're using the `rdfs:label` value here rather than the `em:preferredName`, which allows us to introduce descriptions like "City of Opole" instead of displaying just "Opole" (the preferred name).  The degenerate case, and probably the common case (e.g. when using data extracted automatically from GeoNames), would be where the label and preferred name are the same string.

Are you OK with this approach - i.e. making a distinction between the preferred name and display label?
